src/main/java/org/sqlite/core/NativeDB.java

```diff
@@ -90,11 +90,7 @@ protected synchronized void _open(String file, int openFlags) throws SQLExceptio
     /** @see org.sqlite.core.DB#_exec(java.lang.String) */
     @Override
     public synchronized int _exec(String sql) throws SQLException {
-        logger.atTrace()
-                .setMessage("DriverManager [{}] [SQLite EXEC] {}")
-                .addArgument(() -> Thread.currentThread().getName())
-                .addArgument(sql)
-                .log();
+        logger.trace("DriverManager [{}] [SQLite EXEC] {}", Thread.currentThread().getName(), sql);
```

Comment:
shouldn't we need to check the log level is trace with an `if` to avoid the performance of getting the thread name?

Reply:
Ah thanks for the heads-up I haven't noticed the lambda, I will go through the PR and fix if there are other

Reply:
This is fixed in PR update https://github.com/xerial/sqlite-jdbc/compare/e33f9c8441ffb77d722d8dcddbdc18866bd4588e..221b71a762f9db3a078e30d860c5120f7c74ddfe